spec/jobs/whitehall_document_import_job_spec.rb

```diff
@@ -22,12 +22,18 @@
     )
   end
 
-  it "calls WhitehallImporter::Import" do
+  it "calls WhitehallImporter::Import if the document import is pending" do
     expect(WhitehallImporter::Import).to receive(:call)
                                      .with(whitehall_migration_document_import)
     described_class.perform_now(whitehall_migration_document_import)
   end
 
+  it "does not call WhitehallImporter::Import if the document import is not pending" do
+    expect(WhitehallImporter::Import).not_to receive(:call)
+                                     .with(imported_document_import)
```

Comment:
I'm not sure the `with` is strictly necessary here, since we don't want `WhitehallImporter::Import` to be called at all

Reply:
Fair point 👍